plugins/modules/docker_volume.py

```diff
@@ -214,7 +214,7 @@ def has_different_config(self):
                                         parameter=value,
                                         active=self.existing_volume['Options'].get(key))
         if self.parameters.labels:
-            existing_labels = self.existing_volume.get('Labels', {})
+            existing_labels = self.existing_volume.get('Labels', {}) or {}
```

Comment:
This way it's a bit shorter:
```suggestion
            existing_labels = self.existing_volume.get('Labels') or {}
```